src/interpreter/mod.rs

```diff
@@ -178,7 +179,15 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
             }}
         }
         match *const_val {
-            Float(_f) => unimplemented!(),
+            Float(ConstFloat::F32(f)) => {
+                let i = unsafe { transmute::<_, u32>(f) };
```

Comment:
In the spirit of paranoia, I'd rather avoid adding transmutes and instead use the byteorder crate's `{read,write}_f{32,64}` methods (with matching methods on `Memory`, I suppose).